### PR TITLE
switch node names to machine ID

### PR DESCRIPTION
**Pull request by benwaffle**
6pn as node name causes us some problems

https://github.com/superfly/postgres-stub/issues/5

TODO:
- [x] make sure `fly image update` works from 6pn node names to machine ID node names 
- [x] fix a few instances `[%s]:5500` which assume the node name is a 6pn

### CODE REVIEW

**Comment by davissp14** [comment]
This one is fairly hidden, but wanted to make sure you had eyes on it: 
https://github.com/superfly/flyctl/blob/master/internal/command/postgres/postgres.go#L194

File: internal/flypg/pg.go [hunk 1]
```diff
@@ -606,3 +606,11 @@ func diskSizeInBytes(dir string) (uint64, error) {
 	}
 	return stat.Blocks * uint64(stat.Bsize), nil
 }
+
+func (*PGConfig) reload(ctx context.Context) error {
```
Comment (davissp14) [inline internal/flypg/pg.go:610]: This totally works, but you can also issue a reload using sql:  https://github.com/fly-apps/postgres-flex/blob/master/internal/flypg/admin/admin.go#L324-L329

File: internal/flypg/zombie.go [hunk 2]
```diff
@@ -117,7 +117,8 @@ func TakeDNASample(ctx context.Context, node *Node, standbys []Member) (*DNASamp
 		sample.totalActive++
 
 		// Record conflict when primary doesn't match.
-		if primary.Hostname != node.MachineID && primary.Hostname != node.PrivateIP {
+		// We're checking PrivateIP here for backwards compatibility
```
Comment (davissp14) [inline internal/flypg/zombie.go:120]: Do we need to remain backwards compatible here?

Comment (benwaffle) [reply internal/flypg/zombie.go:120]: I think so, during `fly image update`, when we do a rolling upgrade of the nodes

**Pull request merged**